web/src/types.ts

```diff
@@ -67,4 +77,5 @@ export type LogLine = {
   text: string
   manifestName: string
   level: string
+  fields?: Fields | null
```

Comment:
will this break React caching? see the comment on line 73 (a couple lines above this)

it might make sense to just put progressID and buildEvent directly on the LogLine

Reply:
Oops, I read this comment and subsequently forgot about it. I'll put buildEvent directly on the LogLine.